frontend/src/metabase/components/__mocks__/ExplicitSize.tsx

```diff
@@ -0,0 +1,10 @@
+const ExplicitSize = () => (ComposedComponent: any) => {
+  const WrappedComponent = (props: any) => (
+    <ComposedComponent width={1000} height={1000} {...props} />
+  );
+
+  return WrappedComponent;
+};
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
nit: we can remove this eslint rule from `__mocks__`

Reply:
It's a typescript file, so eslint will actually complaint :(

Reply:
I meant we can adjust eslint config to exclude such checks from `__mocks__`